src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.ipp

```diff
@@ -376,7 +368,13 @@ template <class ImplClass>
 void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::BackgroundEventLoopTaskMain(void * arg)
 {
     ChipLogDetail(DeviceLayer, "CHIP background task running");
-    static_cast<GenericPlatformManagerImpl_FreeRTOS<ImplClass> *>(arg)->Impl()->RunBackgroundEventLoop();
+    GenericPlatformManagerImpl_FreeRTOS<ImplClass> * platformManager =
+        static_cast<GenericPlatformManagerImpl_FreeRTOS<ImplClass> *>(arg);
+    platformManager->Impl()->RunBackgroundEventLoop();
+    vTaskDelete(NULL);
+    vQueueDelete(platformManager->mBackgroundEventQueue);
```

Comment:
If the resources are initialized in `_InitChipStack()`, they should be released in `_Shutdown()`.

Reply:
Yes, done